dev/Scroller/TestUI/ScrollerBringIntoViewPage.xaml.cs

```diff
@@ -295,26 +366,52 @@ private void BtnStartBringIntoView_Click(object sender, RoutedEventArgs e)
 
         private void CmbNestingCombination_SelectionChanged(object sender, SelectionChangedEventArgs e)
         {
-            if (outerScroller == null || outerScrollViewer == null || outerScroller2 == null) return;
+            if (outerScroller == null ||
```

Comment:
The only case when these would be null is before the page has been fully initialized yet - correct? I think there is an auto-generated 'isInitialized' field that you can check for this.

Reply:
Couldn't find any such autogenerated field. So just left it as is.

Reply:
The field I was thinking of is called "_contentLoaded" - however it looks like it is set to true **before** calling LoadComponent instead of after, so it wouldn't be useful here.